examples/autocomplete-suggestions/pages/index.js

```diff
@@ -0,0 +1,44 @@
+import fetcher from '../libs/fetcher'
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxPopover,
+  ComboboxList,
+  ComboboxOption
+} from '@reach/combobox'
+
+import useSWR from 'swr'
+
+export default () => {
+  const [searchTerm, setSearchTerm] = React.useState(null)
+  const { data: countries = [], isValidating } = useSWR(
+    () => (searchTerm ? `/api/suggestions?value=${searchTerm}` : null),
+    fetcher
+  )
+
+  function handleSearchTermChange(event) {
+    setSearchTerm(event.target.value)
+  }
+
+  return (
+    <div style={{ textAlign: 'center' }}>
+      <h1>Country Search</h1>
+      <Combobox>
+        <ComboboxInput
+          className="country-search-input"
+          onChange={handleSearchTermChange}
+          aria-label="Countries"
+        />
+        {!isValidating && countries.length > 0 && (
```

Comment:
```suggestion
        {countries && countries.length > 0 && (
```

There're some cases that `isValidating` is true but we have the data (e.g.: when focusing on the tab).